doc/sphinx/compiling.rst

```diff
@@ -5,21 +5,22 @@ Cantera Compilation Guide
 *************************
 
 .. toctree::
-   :hidden:
 
    SCons Configuration Options <configuring>
 
 This guide contains instructions for compiling Cantera on the following
 operating systems:
 
-* Linux
+* `Linux`_
 
-  * Ubuntu 12.04 LTS (Precise Pangolin) or newer; 16.04 LTS (Xenial Xerus) or
-    newer is recommended
-  * Debian 7.0 (Wheezy) or newer; 8.0 (Jessie) or newer is recommended
-  * Fedora 20 or newer; 22 or newer is recommended
+  * :ref:`sec-ubuntu-debian-reqs`
+  * :ref:`sec-fedora-reqs`
+  * :ref:`sec-opensuse-reqs`
+
+* `Windows`_
+
+  * :ref:`sec-windows-reqs`
 
-* Windows 7 or newer (32-bit or 64-bit versions)
 * OS X 10.9 (Mavericks) or newer; 10.10 (Yosemite) or newer is recommended
```

Comment:
Given that we recommend people use Homebrew Python to install Cantera for non-developers, should we make the same recommendation for developers? Right now, the compiling docs suggest using the system Python, and given the changes to how system directories are handled in recent versions of OS X (macOS), I'm leery of recommending people use `sudo pip install ...`.